tests/e2e/medic-collect.js

```diff
@@ -36,32 +36,31 @@ describe('medic-collect', () => {
     ]));
 
   /*  afterAll(() =>
-    Promise.all([ 'form:my_app_form', 'form:my_collect_form' ].map(id =>
+    await Promise.all([ 'form:my_app_form', 'form:my_collect_form' ].map(id =>
```

Comment:
Instead of modifying the commented-out code, please remove it if it is not necessary.